### ISSUE TITLE
Long distance rides 

### ISSUE THREAD

**Issue by Stretch379** [TeleportXYZ/TRIP-Guides#49]
I personally am a big fan of the longer trips, such as DSW to Houston or Shreveport or Oklahoma City etc. it would be nice if we could integrate a filter for the long trips ### What's missing from the TRIP Guides

**Comment by hwhelchel** [comment]
@Stretch379 Gary let's modify this to be a section on Rider Pickup and Drop Off Limits. 

In other words, what limits exist if any on where riders can be picked up or dropped off?

@abhilash710 what limits if any will exist on Rider Pickup and Drop Off in the Genesis City or in Texas in general?

**Comment by Stretch379** [comment]
We can integrate the same process that Hitch has, where it’s a public well lit place for either drop off or pick ups if they’re not comfortable with the home address

**Comment by hwhelchel** [comment]
Yes, possibly! I'm referring to something more basic than that. We'll be launching first in one city in Texas. Given that, there will be almost like a fence around the city setting limits on where rides can take place. Often called a geofence.

So this limit won't have to do with dropping off / picking people up a block or two from their home address. I'm more referring to the long distances you like to drive. It's likely the case that there won't be long distance drives to start out. Not until we at least have a large contiguous area of the state / country operating on the network.

**Comment by Alejandro1954-ops** [comment]
> Yes, possibly! I'm referring to something more basic than that. We'll be launching first in one city in Texas. Given that, there will be almost like a fence around the city setting limits on where rides can take place. Often called a geofence.
> 
> So this limit won't have to do with dropping off / picking people up a block or two from their home address. I'm more referring to the long distances you like to drive. It's likely the case that there won't be long distance drives to start out. Not until we at least have a large contiguous area of the state / country operating on the network.

@hwhelchel difficult to say as each driver has their own preference. Ideally, nationwide or even cross border (USA, Canada, Mexico) lol. 

As for giving preference options, ideally we should let the drivers adjust their maps to allow their preferred pickup radius. Realistically, for both the devs and passengers, maybe it can be something like Lyft and Uber already have: 30,60,90 mile radius options. 

**OR**

If the passenger ride falls outside of the 30,60,90 mile range the passenger will have to have to pay a long distance fee. This fee will be added for the driver before the passenger confirms the trip start and stop locations (accounting for more than likely the passengerless ride back for the driver)

Anyway, I know this is a post Genesis City launch issue.

**Comment by hwhelchel** [comment]
@Alejandro1954-ops thank you for your contribution.

The TRIP Guides focuses on describing the current state of the system as is. If we start adding a bunch of content around how the system should work or will work in the future, it will just confuse people.

Right now there are polygons we put on the map that technically limit drop off and pick up of drivers. It doesn't and can't take into account driver preferences for longer distance trips at the moment.

**Comment by hwhelchel** [comment]
@abhilash710 can you or someone else on the product side share a bit about how Rider Pickup and Drop Off limits work today in the system?
